langlib/lang.int/src/main/java/org/ballerinalang/langlib/integer/Abs.java

```diff
@@ -32,6 +40,11 @@
 public class Abs {
 
     public static long abs(long n) {
+        if (n <= Long.MIN_VALUE) {
+            throw ErrorCreator.createError(getModulePrefixedReason(INT_LANG_LIB,
```

Comment:
I think we can move this to a constant?

Reply:
But, if we keep it in a constant, the error will be initialised even if there's no error. Wouldn't that be unnecessary?